package.json

```diff
@@ -9,8 +9,8 @@
     "build:mainnet": "node build/build.js --network mainnet",
```

Comment:
This should probably also be updated to use `--env.network` as you removed minimist.